test/message-cache.spec.ts

```diff
@@ -5,6 +5,14 @@ import { MessageCache } from '../src/message-cache.js'
 import * as utils from '@libp2p/pubsub/utils'
 import { getMsgId } from './utils/index.js'
 import type { RPC } from '../src/message/rpc.js'
+import { MessageId } from '../src/types.js'
+
+const toMessageId = (msgId: Uint8Array): MessageId => {
```

Comment:
this will need to be tweaked after #270

Reply:
no, we can use `messageIdToString()` function directly in test